entrypoint.sh

```diff
@@ -17,13 +18,13 @@ if [[ $env == "sit" || $env == "uat" ]]; then
 fi
 
 if [[ "${LAUNCHPAD_TOKEN_SIT}" ]] && [[ $env == "sit" ]]; then
-  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p POCLOUD -e ${env} -t $LAUNCHPAD_TOKEN_SIT
+  umms_updater -d -f cmr/cmr.json -a cmr/${env}_associations.txt -p POCLOUD -e ${env} -t $LAUNCHPAD_TOKEN_SIT -to $timeout
```

Comment:
This is unrelated to your ticket but I noticed our script is hard coding POCLOUD provider here instead of using the input parameter. Can you update that?

Reply:
ok